Makefile

```diff
@@ -1,6 +1,20 @@
 HOSTNAME := https://sw.bakerstreet.io
 DRY_RUN := true
+RUNNER_TAG := ''
+
+.PHONY: test-ubuntu-arm64
+test-ubuntu-arm64:
+	# SHA can be calculated like this:
+    #echo -n "payload=$(cat github-runner-provisioner/test/ubuntu-arm64_payload.json)" | openssl dgst -sha1 -hmac FAKE_TOKEN
```

Comment:
👀 Thank you for adding this comment! Super helpful 🙌

Reply:
My plan is to make the target smarter, so you can pass any token, but I'm not an expert and want to get this out ASAP.